assets/stylesheets/components/_button-form-group.scss

```diff
@@ -24,18 +24,25 @@
   }
 
   .custom-input {
-    display: flex;
+    position: relative;
     justify-content: flex-end;
```

Comment:
Since this is no longer a flex item, is the `justify-content` rule still needed?